### DISCUSSION TITLE
Is there any way to clean records by hour.

### DISCUSSION THREAD

**Discussion by 983183947** [apache/skywalking#11264] [Q&A] [answered]
Is there any way to clean records by hour.

**Comment by wu-sheng** [comment, marked as answer]
No, there isn't.

**Reply by 983183947** [reply]
Then I think there will be problems cleaning up the trace data by myself, whether it is enough to clean up the segment table

**Reply by wu-sheng** [reply]
Don't know what you mean. Most databases are easy to scale, or have alternative for cluster mode.
I can't see any limitation.
If you insist, that is your problem not ours.

**Reply by 983183947** [reply]
skywalking产生的数据太大了，我只想数据保留半天。我查询了下表占用的空间发现基本都是segment这张表占用的，我是否可以直接清理这张表？

**Reply by wu-sheng** [reply]
This is not a Chinese discussion channel. I have no choice to close this as you even don't hear what I said.

Do anything you want, your env is your responsibility
